taichi/runtime/llvm/runtime_module/runtime.cpp

```diff
@@ -1831,6 +1831,7 @@ void taichi_printf(LLVMRuntime *runtime, const char *format, Args &&...args) {
   printf_helper helper;
   helper.push_back(std::forward<Args>(args)...);
   cuda_vprintf((Ptr)format, helper.ptr());
+#elif ARCH_amdgpu
```

Comment:
Add a TODO/FIXME to say that print on AMDGPU isn't supported yet. 

On the other hand, I'd suggest reject `PrintStmt` during `amdgpu_codegen`

Reply:
1. Sure, I will add more comments.
2. `TI_NOT_IMPLEMENTED` will be thrown during `codegen`